containers/RepositoryPullRequestScreen/RepositoryPullRequestScreen.js

```diff
@@ -101,6 +115,36 @@ class RepositoryPullRequestScreen extends PureComponent<Props> {
                     </View>
                 </View>
                 <UIText style={styles.body}>{pullRequest.body}</UIText>
+                <ImageButton
+                  style={styles.button}
+                  onPress={() => this.props.addModal(
+                    <ModalPicker
+                        data={[{ type: 'merge' }]}
+                        renderOption={
+                            ({ item }) => {
+                                return (
```

Comment:
You can omit it.
```js
renderOption={({ item }) => (
    <TouchableOpacity
         key={item.type}
         style={styles.modalContent}
         onPress={() => { // <- remove this arrow function passing
             closeModal();
         }}
    >
        <Text style={styles.modalTitle}>Merge this pull request?</Text>
        <Button
            style={styles.deletePr}
            textStyle={styles.deletePrText}
            onPress={() => this.props.closeModal()} // <- remove this arrow function passing
        >
            Merge
        </Button>
    </TouchableOpacity>
)}

```
But, I do prefer extract this rendering to a separate function.